src/pages/materi-kecakapan-umum/tali-temali.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image'
+import Layout from '../../components/MainLayout'
+import MetaHead from '../../components/MetaHead'
+import ActionNavigation from '../../components/ActionNavigation'
+import BreadcrumbLevel3 from '../../components/Breadcrumb/Level3'
+import ChapterTitle from '../../components/ChapterTitle'
+import SeoText from '../../components/SeoText'
+import { BASE_PATH } from '../../constants'
+import TaliTemaliData from '../../data/tali-temali'
+import { ReportCoreWebVitalsParams, reportCoreWebVitals } from '../../utils/index'
+import Expansion from '../../components/Expansion'
+
+const title = 'Tali temali'
+const desc =
+  'Kumpulan tali - temali'
+const url = BASE_PATH + '/materi-kecakapan-umum/tali-temali/'
+
+function TaliTemali() {
+  return (
+    <Layout>
+      <>
+        <MetaHead title={title} desc={desc} url={url} />
+        <BreadcrumbLevel3 text="Tali Temali" href={url} />
+        <ChapterTitle subTitle="Tali Temali" title="Materi Kecakapan Umum" />
+        <ActionNavigation title={title} text={desc} url={url} />
+
+        <div className="text-left">
+          <div className="mt-4">
+            {TaliTemaliData.title}
```

Comment:
This part seems weird, can we just delete this title?
We already have a ChapterTitle

Reply:
Hi mas @mazipan, I'll fix this in the next feature

Reply:
Cool, thanks